walletrpc/service.proto

```diff
@@ -118,6 +118,23 @@ message TreeState {
     string orchardTree = 6; // orchard commitment tree state
 }
 
+enum ShieldedProtocol {
+    SAPLING = 0;
+    ORCHARD = 1;
+}
+
+message GetSubtreeRootsArg {
+    uint64 startIndex = 1;                  // Index identifying where to start returning subtree roots
+    ShieldedProtocol shieldedProtocol = 2;  // Shielded protocol to return subtree roots for
+    uint64 limit = 3;                       // Maximum number of entries to return, or 0 for all entries.
+}
+message SubtreeRoot {
+    uint64 index = 1;                       // The index of the subtree. Each subtree containx 2^16 leaves.
+    string rootHash = 2;                    // The Merkle root of the subtree.
+    string completingBlockHash = 3;         // The hash of the block that completed this subtree.
```

Comment:
```suggestion
    bytes rootHash = 2;                     // The Merkle root of the subtree.
    bytes completingBlockHash = 3;          // The hash of the block that completed this subtree.
```
`bytes` is a better data type than `string` for binary data (although we do use `string` in several places already that should be `bytes`, and that would be very difficult to change, but going forward, we should use `bytes`).

Reply:
Yup, let's use `bytes`